### PR TITLE
Self management

### LINKED ISSUE #22: Web: Manage User Info
Provide endpoints that work with the actions for users provided in `core`:

- change username
- change email
- change org information
- update password

Obviously, this needs to correspond to the authenticated user.

### PR DESCRIPTION
**Pull request by vforgione**
users can change:

- name
- email address
- password
- org info

i also fixed a stupid mistake i made in guardian pinning user resources to the email address rather than their id.

closes #22

### CODE REVIEW

File: lib/plenario2_web/controllers/meta_controller.ex [hunk 1]
```diff
@@ -59,7 +59,6 @@ defmodule Plenario2Web.MetaController do
 
   defp create_reply({:ok, meta}, conn) do
     conn
-    |> put_status(:created)
```
Comment (vforgione) [inline lib/plenario2_web/controllers/meta_controller.ex]: TIL doing this breaks the redirect

File: lib/plenario2_auth/guardian.ex [hunk 2]
```diff
@@ -3,11 +3,11 @@ defmodule Plenario2Auth.Guardian do
   alias Plenario2Auth.UserActions
 
   def subject_for_token(user, _claims) do
-    {:ok, user.email_address}
+    {:ok, user.id}
```
Comment (vforgione) [inline lib/plenario2_auth/guardian.ex]: pinning to the email was a bad idea

File: lib/plenario2_web/templates/layout/app.html.eex [hunk 3]
```diff
@@ -9,6 +9,23 @@
 
     <title>Plenario</title>
     <link rel="stylesheet" href="<%= static_path(@conn, "/css/app.css") %>">
+    <style>
```
Comment (vforgione) [inline lib/plenario2_web/templates/layout/app.html.eex]: i couldn't figure out how to effect this via `app.css` so i rammed it in here.

basically, i hate being forced into a 768px view port on a modern laptop screen. it just makes everything compressed and look like shit.

File: .gitignore [hunk 4]
```diff
@@ -11,6 +11,7 @@ erl_crash.dump
 npm-debug.log
 
 # Static artifacts
+node_modules/
```
Comment (HeyZoos) [inline .gitignore]: Huh, why are `node_modules` making it outside of the `assets/` folder?

**Pull request merged**